### ISSUE TITLE
Add todo format check to travis.yml

### ISSUE THREAD

**Issue by longtimeago** [jcabi/jcabi-ssh#49]
It would be good to check puzzle format with travis.
Need to update `travis.yml` with:
```
before_install:
  - set -e
  - pdd --source=$(pwd) --file=/dev/null
```

**Comment by dmarkov** [comment]
@yegor256 please do something about it, see [par.21](http://at.teamed.io/policy.html#21)

**Comment by yegor256** [comment]
@longtimeago fixed in e73df79 thanks
